v19.2/managed-connect-to-your-cluster.md

```diff
@@ -7,77 +7,106 @@ build_for: [managed]
 
-## Before you begin
+## Step 1. Authorize your network
 
-1. On the machine from which you want to connect to the cluster, create a `certs` directory. The `ca.crt` file that you download later will go here.
+Managed CockroachDB requires you to authorize the networks that can access the cluster to prevent denial-of-service and brute force password attacks:
 
-    Any client connecting to the cluster will need access to the `ca.crt` file.
+- In a development environment, you need to authorize your application server’s network and your local machine’s network. If you change your location, you need to authorize the new location’s network, or else the connection from that network will be rejected.
+- In a production environment, you need to authorize your application server’s network.
 
-2. Make sure you have a SQL user and password to use in your connection string.
+Use the Console to authorize networks:
 
-    For details on creating additional users, see [User Management](managed-authorization.html#use-the-console).
+1. Navigate to your cluster's **Networking** page.
 
-## Authorize your network
+    The **Networking** page displays a list of authorized networks (i.e., an IP network whitelist) that can access the cluster.
 
-Before you connect to your cluster, you need to authorize your network (i.e., whitelist the public IP address of the machine). Otherwise, connections from this machine will be rejected.
+    <img src="{{ 'images/v19.1/managed/networking.png' | relative_url }}" alt="Networking page" style="border:1px solid #eee;max-width:100%" />
 
-Once you are [logged in](managed-sign-up-for-a-cluster.html#sign-in), you can use the Console to authorize your network:
+2. Check if the current network has been authorized. If not, proceed with the following steps.
 
-1. Navigate to your cluster's [Networking](managed-networking-page.html) page.
-2. Click the **Add Network** button in the top right corner.
+3. Click the **Add Network** button in the top right corner.
 
     The **Add Network** modal displays.
 
-    <img src="{{ 'images/v19.2/managed/add-network-modal.png' | relative_url }}" alt="Add network" style="border:1px solid #eee;max-width:100%" />
+    <img src="{{ 'images/v19.1/managed/add-network-modal.png' | relative_url }}" alt="Add network" style="border:1px solid #eee;max-width:100%" />
 
-3. Enter the public IP address of the machine in the **Network** field.
+4. Enter the public IPv4 address of the machine in the **Network** field.
 
-    The IP address should be written in Classless Inter-Domain Routing (CIDR) notation. For example:
+    You can use `0.0.0.0/0`, which allows all networks. Use this with caution; anybody who uses your password will be able to access the database, and your cluster will be more exposed if there's ever a security bug. The firewall is an extra layer of defense.
```

Comment:
I think I prefer the "prevent denial-of-service and brute force password attacks" rationale. But either way, it seems that we should stick to one phrasing for all whitelist warnings.